docs/SetupDatabaseForAKS.md

```diff
@@ -0,0 +1,75 @@
+# Set up a database for Scalar DB/Scalar DL deployment in AKS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment in AKS.
+
+## Cosmos DB
+
+In this section, you will create a Cosmos DB account.
+
+### Requirements
+
+* You must create a Cosmos DB account with `Core(SQL)` API.
+* You must change Cosmos DB `Default consistency` to `Strong`.
+* You must use Capacity mode as `Provisioned throughput`.
+
+### Recommendations
+
+* You should configure the backup policy as `Continuous` for Point-in-time-restore (PITR).
+* You should configure the service endpoint based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-configure-vnet-service-endpoint) to restrict access to Cosmos DB for production.
+
+### Steps
+
+* Create an Azure Cosmos DB account based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account) with the above requirements.
+
+### Scale Performance
```

Comment:
`Scale Performance` is the same level as `Requirements`, `Recommendations`, and `Steps`?
Is this not required?
If this should be one of the steps, it should go in the steps.

Reply:
Then, the existing structure looks weird because they shouldn't be the same level, right?
Please update it in another PR.

Reply:
> Then, the existing structure looks weird because they shouldn't be the same level, right?
Please update it in another PR.

Regarding updating structure in existing [SetupDatabase.md](https://github.com/scalar-labs/scalar-kubernetes/blob/master/docs/SetupDatabase.md), I think we can remove that document once we merge this document and the one for EKS.
I will update the structure added in this PR.